latch_cli/centromere/ctx.py

```diff
@@ -120,6 +120,11 @@ def __init__(
                                 image_name=self.task_image_name(entity.name),
                                 pkg_dir=entity.dockerfile_path.parent,
                             )
+                if hasattr(self, "workflow_name"):
```

Comment:
`click.secho(..., fg="red")` and then `raise click.Exit(1)` instead of throwing a valueerror

Reply:
see error messaging on latch cp for examples of what i mean